docs/docs/reference/python-sdk/weave/trace/weave.trace.weave_client.md

```diff
@@ -88,8 +88,8 @@ Add a cost to the current project.
 **Args:**
  
  - <b>`llm_id`</b>:  The ID of the LLM. eg "gpt-4o-mini-2024-07-18" 
- - <b>`prompt_token_cost`</b>:  The cost of the prompt tokens. eg .0005 
- - <b>`completion_token_cost`</b>:  The cost of the completion tokens. eg .0015 
+ - <b>`prompt_token_cost`</b>:  The cost per prompt token. eg .0005 
```

Comment:
2nd change

Reply:
Is this just the autogen change corresponding to change #3? To simplify review it could be helpful to split your changes out from codegen changes in separate commits.